lib/faraday/options/env.rb

```diff
@@ -203,3 +206,5 @@ def self.member_set
     end
   end
 end
+# rubocop:enable Metrics/BlockLength
+# rubocop:enable Metrics/ModuleLength
```

Comment:
Lot of linter wrangling here, LMK if using the config file is preferable.

Reply:
Yeah I believe we already have `Metrics/BlockLength` and `Metrics/ModuleLength` in `.rubocop_todo.yml` so I'd rather update the values there instead 👍